`legacy-esm.js` file not included in NPM package for `@whereby.com/core`

# Issue

When installing and using `@whereby.com/browser-sdk@3.9.11` in my Vite frontend project, I encounter the following error when attempting to run the application:

```
✘ [ERROR] [plugin vite:dep-pre-bundle] Failed to resolve entry for package "@whereby.com/core". The package may have incorrect main/module/exports specified in its package.json.
....
The plugin "vite:dep-pre-bundle" was triggered by this import

    node_modules/@whereby.com/browser-sdk/dist/react/index.esm.js:3:1298:
      3 │ ...rentMicrophoneDeviceId, setCurrentSpeakerDeviceId, selectAllClientViews, selectNumParticipants } from '@whereby.com/core';
        ╵                                                                                                          ~~~~~~~~~~~~~~~~~~~
```

# Investigation/Details

The installed version of `@whereby.com/core` is `0.29.0`, and checking that version on [publint](https://publint.dev/@whereby.com/core@0.29.0) it shows that the `dist/legacy-esm.js` file does not exist in the published package.

Digging a little further, I cloned and built the repository and confirmed that `legacy-esm.js` is indeed being generated, but the `packages/core/package.json` file does not list `*.js` in the `files` property.

------

I'd love to use this in my project but this is a blocker to moving forward, if you have some bandwidth soon it would be great if this could be looked at.  I can also provide a corresponding PR if that would help move things along.  Thanks!

--- a/.changeset/loud-donkeys-exercise.md
+++ b/.changeset/loud-donkeys-exercise.md
@@ -1,0 +1,5 @@
+---
+"@whereby.com/core": patch
+---
+
+Add .js files to the files array in package.json

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -17,6 +17,7 @@
         "access": "public"
     },
     "files": [
+        "dist/*.js",
         "dist/*.cjs",
         "dist/*.mjs",
         "dist/*.d.ts",

